src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -257,6 +257,13 @@ private static string GetClassificationForLocal(ILocalSymbol localSymbol)
 
         private static string GetClassificationForMethod(IMethodSymbol methodSymbol)
         {
+            if (methodSymbol.MethodKind == MethodKind.Constructor)
```

Comment:
The destructor declaration is classified syntactically in the `ClassifierHelpers`. Since the destructor cannot be invoked, there is no semantic classification necessary. I added a comment.